src/platform/datapath_raw_xdp.c

```diff
@@ -1570,6 +1560,27 @@ CxPlatXdpTx(
     }
 }
 
+_IRQL_requires_max_(DISPATCH_LEVEL)
+void
+CxPlatDpRawTxEnqueue(
+    _In_ CXPLAT_SEND_DATA* SendData,
+    _In_ BOOLEAN InlineHint
+    )
+{
+    XDP_TX_PACKET* Packet = (XDP_TX_PACKET*)SendData;
+    XDP_QUEUE* Queue = Packet->Queue;
+    if (Queue->Worker->ProcIndex == CxPlatProcCurrentNumber()) {
```

Comment:
Maybe, instead of always querying proc, just use compile-time `#ifdef QUIC_USE_EXECUTION_CONTEXTS`.